### ISSUE TITLE
Localizing

### ISSUE THREAD

**Issue by enterprisey** [WPAFC/afch-rewrite#138]
@94rain, I noticed you had a translated fork of this script. It shouldn't be that difficult to just have a large file with translations of all the messages. Would you be interested in something like that?

**Comment by 94rain** [comment]
Yes, I think that's a good idea. How would you plan to implement that?

**Comment by enterprisey** [comment]
Fortunately, this script has a build step, so perhaps we could have a new directory with one json file per language. Do you think we would need extra handling for pluralization, gender, or other conjugations? (like https://translatewiki.net/wiki/Plural)

**Comment by 94rain** [comment]
I think it is not needed for en / zh in the current state. Something like `oneComment ? 'an' : ''` are already handled in the script.

**Comment by Jeeputer** [comment]
@enterprisey Hi. I am trying to have a Persian version of the afch-rewrite. Do you still plan to create the file you mentioned here? Thanks.

**Comment by robertsky** [comment]
@enterprisey, for certain messages like those we leave on user talk pages, can we offload to the Template namespace? this will reduce the need to translate or customise messages for the different language wikis.
